### PR TITLE
Falsey zeroes

### LINKED ISSUE #74: False-y values in a numeric variable shouldn't be imputed to 0
Right now "" is being drawn as a 0 - it should probably be null instead.

### PR DESCRIPTION
**Pull request by samussiah**
# Issues
Closes #74

### CODE REVIEW

File: src/chart/draw/consolidateData/setDomain.js [hunk 1]
```diff
@@ -5,9 +5,12 @@ export default function setDomain(axis) {
     const otherAxis = axis === 'x' ? 'y' : 'x';
```
Comment (jwildfire) [inline src/chart/draw/consolidateData/setDomain.js]: thanks for adding this. very helpful.

File: src/chart/draw/consolidateData/transformData.js [hunk 2]
```diff
@@ -14,108 +15,51 @@ import { time, set, extent, merge } from 'd3';
...
-        });
-    }
-    if ((config.y.type === 'linear' || config.y.type === 'log') && config.y.column) {
-        raw = raw.filter(f => {
-            return mark.summarizeY !== 'count' && mark.summarizeY !== 'percent'
-                ? +f[config.y.column] || +f[config.y.column] === 0
-                : f;
-        });
-    }
+    const cleaned = cleanData.call(this, mark, raw);
```
Comment (jwildfire) [inline src/chart/draw/consolidateData/transformData.js]: Careful with all the refactoring - this code is going to be understandable soon! 😆

File: test/test.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+//remove falsey values from data
```
Comment (jwildfire) [inline test/test.js]: is there a better name than test.js?

Comment (samussiah) [reply test/test.js]: Maybe newUnitTest.js?  It's there to avoid running all tests, just the one currently being worked on.  I'll remove it at the end.

**Pull request merged**